CHANGELOG.md

```diff
@@ -1,3 +1,10 @@
+# dbt_jira v0.9.0
+
+PR [#66](https://github.com/fivetran/dbt_jira/pull/66) incorporates the following updates:
+## 🚨 Breaking Changes 🚨
+- Flipped column aliases `sum_close_time_seconds` and `sum_current_open_seconds` of intermediate model `int_jira__user_metrics.sql`.
+- This ensures that downstream model `jira__user_enhanced.sql` calculates columns `avg_age_currently_open_seconds` and `avg_close_time_seconds` correctly.
```

Comment:
Since this next breaking change is going to have a number of PRs (two for right now at least) we should probably do the direct link of the PR next to the bullet.